### ISSUE TITLE
Design mockup of Delete a reply 

### ISSUE THREAD

**Issue by kevimuri24** [zurichat/zc_plugin_channels#305]

**Comment by Ebuka-Victor** [comment]
https://www.figma.com/file/h9nI4CEnyBqVAx5yljc5QD/Delete-a-reply-(Channels-plugin)?node-id=0%3A1

Slack Username: VictorOnwuelu
